http/data_source.go

```diff
@@ -62,7 +62,7 @@ func dataSourceRead(d *schema.ResourceData, meta interface{}) error {
 
 	resp, err := client.Do(req)
 	if err != nil {
-		return fmt.Errorf("Error during making a request: %s", url)
+		return fmt.Errorf("Error during making a request: %s", err)
```

Comment:
@whatabout-bob does `err` include the url? 🤔Happy to get the extra info added, my gut is telling me though you meant:
```
return fmt.Errorf("Error making a request: %s: %s", url, err)
```

I also suggest omitting `during` as it is an awkward error message while we are here!

Reply:
Hi @appilon, thanks for the feedback!  Yes, through my testing I've verified that the output from `err` does include the request URL along with any error message received.  I built this as a custom module and tested it with a failing URL, and got output similar to the following: 

`Error during making a request: Get https://my.url.com/api/some/data: dial tcp 10.0.0.23:443: i/o timeout`

Good suggestion on the error messaging, will make that update.